CMake/Modules/InstallHTMLDOC.cmake

```diff
@@ -0,0 +1,30 @@
+# create CMakeLists.txt for external project
+# required variables:
+## HTMLDOC_INSTALL_URL - from where to download file (zip/tarball)
+## HTMLDOC_INSTALL_DIR - directory where helper files will be stored such as CMakeLists.cmake file makefile
+## HTMLDOC_SOURCE_DIR - direcotry which will contain unpacked files from server
```

Comment:
Is it REALLY necessary to set these variables out of this file? Its better to put this setting here since you can find it easier then in the already huge CMakeLists.txt